Update AG_CONEX.cpp

hjunkes opened the pull request:
FCL200 has its own built-in controller (positioner) and you can address the positions directly in mm. The commands FRM? and FRS? are used for compatibility reasons, but lead to confusion in the driver here (wrong MRES etc.).

hjunkes committed:
Update AG_CONEX.cpp

FCL200 has its own built-in controller (positioner) and you can address the positions directly in mm.
The commands FRM? and FRS? are used for compatibility reasons, but lead to confusion in the driver here (wrong MRES etc.).

MarkRivers commented:
@hjunkes there is a typo in your file, the line begins with a "+" character, so it fails to compile.
```
+   stepSize_ = 1; // Positions in mm on this stage
```

hjunkes committed:
Update AG_CONEX.cpp

Unfortunately I inserted the lines by hand in the editor of github and dragged in a '+' which of course does not belong there. Sorry
